integration/terraform/ec2/README.md

```diff
@@ -37,8 +37,45 @@ for how to easily generate a new policy.
         "ec2:ModifyInstanceAttribute",
         "ec2:RunInstances",
         "ec2:TerminateInstances",
+        "s3:ListBucket",
+        "s3:GetObject",
+        "s3:GetObjectAcl",
```

Comment:
Does the Integration Test need this though? Correct me if I am wrong since the Acl is more public permission and GetObject is more for private which we are using the private s3.